contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,169 @@
+
+Here is a list of seasonality periods to be mindful of:
+
+* Holiday seasons: e.g., Christmas and New Year's, July and August summer vacations.
+* Fiscal year-end: Companies typically review budgeting decisions and contract renewals during this period.
+* Seasonal sales cycles: Some industries, like retail or agriculture, have seasonal sales cycles. Be aware of these cycles if your target customers operate in such industries.
+
+## 7. Testing an unclear hypothesis
+
+Having a clear definition of what you are testing and why you are testing it will help you determine which metrics to measure and analyze. To understand this better, let's look at an example of a bad hypothesis:
+
+> Bad Hypothesis: Changing the color of the "Proceed to checkout" button will increase purchases.
+
+This is bad since it's unclear why we're testing this change and why we expect it to increase purchases. As a result, it's not clear what we need to measure in this experiment. Do we only need to count button clicks? Or is there something else we need to measure?
+
+Here's an improved version:
+
+> Good hypothesis: User research showed that users are unsure of how to proceed to the checkout page. Changing the button's color will lead to more users noticing it and thus more people will proceed to the checkout page. This will then lead to more purchases.
+
+It's now clear that we need to measure the following: 
+* Button clicks, to show if the change in color leads to more people noticing the button
+* Number of purchases, since more people arriving at the checkout page should mean more purchases
+
+This also makes it easier to investigate the experiment results, especially when they are not what we expected. For example:
+* If the color change didn't increase button clicks, the page may have a different issue.
+* If the number of button clicks increased but the number of purchases did not, there may be an issue with the checkout page.
+
+## 8. Over relying on A/B tests for decision-making
+
+Not everything that can be measured matters. Not everything that matters can be measured. It's important to remember that there can be other reasons for shipping things besides for metric changes, such as solving user pain points or creating enjoyable user experiences.
+
+[Raquel](/handbook/company/team), one of our growth engineers here at PostHog, shares an example:
+
+> We ran an experiment on our sign-up page to make our social login buttons more prominent (e.g., "Sign up with Google" and "Sign up with GitHub") instead of signing up with email and password. While more people signed up using Google and Github, overall sign-ups didn't increase and nor did activation. 
+> 
+> Ultimately, we decided to ship the change since we felt that social login lower frictions and provides a better user experience.
+
```

Comment:
@raquelmsmith is this accurate?

Reply:
Yes, though I'm not sure this is the best example for this point. A better example is a strategic change - we really like to A/B test everything that we can, but some things especially around company strategy (which are often times _bets_ more than anything) just have to be committed to without proper data. We're doing this (as a company, but growth is heavily involved in the switch due to the reliance on billing) as we move from a plans-based billing construct to a product-based one. 

Currently/in the past people would subscribe to a plan and get all features. But this made less sense as our product offering grew, so we're changing a lot around how our billing works to go for a multi-product strategy, where each product can be sub'd to separately. This makes sense but is also a bet because technically we could make things work with a plan-based construct. But, it's hard to test this, and since it's strategic it can't really be tested very well (strategy is a bet you make about the future, not just about today) so we don't really have tests lined up for this change specifically (though we will be testing parts of it, like the new UI to make conversion doesn't take a dive).

Reply:
Thanks! I think I'll leave this example, since its relatively easy to explain and I think quite a common scenario. There are ton of reasons to ship things that arent AB tests. Perhaps I should add a list here of reasons why you may ship without AB tests